service/src/main/java/aero/minova/cas/service/mdi/XMLMenu.xsd

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
In Zukunft (spätestens, wenn wir hier etwas ändern, sollten wir es in das XSD-Projekt auslagern.
Ich denke, man kann es erstmal tolerieren, wenn es im CAS drinnen ist.